state/mysql/mysql.go

```diff
@@ -341,7 +341,28 @@ func (m *MySQL) deleteValue(req *state.DeleteRequest) error {
 // BulkDelete removes multiple entries from the store
 // Store Interface.
 func (m *MySQL) BulkDelete(req []state.DeleteRequest) error {
-	return m.executeMulti(nil, req)
+	m.logger.Debug("Executing BulkDelete request")
+
+	tx, err := m.db.Begin()
+	if err != nil {
+		return err
+	}
+
+	if len(req) > 0 {
+		for _, d := range req {
+			da := d // Fix for goSec G601: Implicit memory aliasing in for loop.
+			err = m.Delete(&da)
```

Comment:
Not sure whether we can use &req[i] here.

Reply:
You mean, instead of `da := d`? Both of them can fix it IMO but I used this pattern for consistency with our codebase.

Reply:
/cc @daixiang0